src/app/app.html

```diff
@@ -34,6 +34,12 @@ <h3>CodeMirror Mode PromQL</h3>
   <option selected value="offline">Offline</option>
   <option value="prometheus">Prometheus</option>
 </select>
+<br>
+<label for="fullLanguage">choose if you want to use the complete PromQL language</label>
+<select name="fullPromQLLanguage" id="fullLanguage">
```

Comment:
How about something like

```
Language to complete:
* Full PromQL
* Metric names
```

...because otherwise it's unclear what "no" means.

Reply:
sounds good